lib/payloadAction.ts

```diff
@@ -21,6 +22,10 @@ export const  handlePayloadActions = async (app: IApp, read: IRead,  modify: IMo
             const targetDepartment: string = await getAppSettingValue(read, AppSetting.FallbackTargetDepartment);
             if (actionName) {
                 if (actionName === ActionIds.PERFORM_HANDOVER) {
+                    if (!targetDepartment) {
+                        console.error(Logs.EMPTY_HANDOVER_DEPARTMENT);
+                        return;
```

Comment:
User Experience improvement:  In the case of ActionIds and EndpointActionNames, we also want to still trigger the "Service Unavailable" message (configured in the DF.App UI).

Reply:
UseCases:
1. RC server supports handover:  Send the "service unavailable message" if handover department not configured. DEF: our backend sends RC a handover action. 
2. RC server does not support handover:  Do not send the "service unavailable message".